.github/workflows/dispatch_turbine_go_dependency.yml

```
@@ -1,6 +1,6 @@
 name: Remote Dispatch Turbine Go Dependency Responder
 
-on: [repository_dispatch]
+on: [workflow_dispatch,repository_dispatch]
```

Could we do the following to trigger the workflow automatically on push? What do you think?

```suggestion
on: 
  workflow_dispatch
  repository_dispatch
  push:
    branches:    
      - 'raul/fix-dispatch-event'
```

---

Leaving `workflow_dispatch` out for now. Testing on push